sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Rollback-update-student-assessment-status.sql

```diff
@@ -0,0 +1,11 @@
+/*
+ Rollback assessments status updates.
+ The workflow is already changing the status to 'In progress' and 'Completed',
+ anything different can be considered as the default column value 'Submitted'.
+ */
+UPDATE
+  sims.student_assessments
+SET
+  student_assessment_status = 'Submitted'
+WHERE
```

Comment:
The rollback may be just setting back the default value to column ('Submitted') without condition.

Reply:
It can, but it would be changing possible 'In progress' and 'Completed' statuses already set by the previous ticket already merged.
Between keeping and removing the where both would not be able to put the records back to their original state.
Based on the fact that neither would be perfect I just picked one.